src/app/shared/components/partials/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface IButtonProps {
+  title: string,
+  customClass: string,
+}
+
+const Button: React.FC<IButtonProps> = ({ title, customClass }) => {
+  return (
+    <button className={customClass ? `btn ${customClass}`: 'btn'}>
```

Comment:
Rank B: ko cần if else ở những chỗ này em, `customClass` rỗng thì nó rỗng thôi, chứ ko cần phải if else để in ra mỗi `btn`